src/frontend/src/app/myprofile/myprofile.component.ts

```diff
@@ -23,6 +28,9 @@ export class MyProfileComponent {
       medium: 10,
       hard: 10,
     } as PlayerData;
+    this.playerStats = authService.getPlayerData();
```

Comment:
Can make this variable local to function instead since it isn't used

Reply:
nevermind Youssef mentioned already

Reply:
Made playerStats a local variable instead.